system-contracts/contracts/EvmInterpreterFunctions.template.yul

```diff
@@ -310,11 +301,7 @@ function getDeployedBytecode() {
 
 function consumeEvmFrame() -> passGas, isStatic, callerEVM {
     // function consumeEvmFrame() external returns (uint256 passGas, bool isStatic)
-    // TODO: Unhardcode selector
-    mstore8(0, 0x04)
-    mstore8(1, 0xc1)
-    mstore8(2, 0x4e)
-    mstore8(3, 0x9e)
+    mstore(0, 0x04C14E9E00000000000000000000000000000000000000000000000000000000)
```

Comment:
@jrchatruc Is there a possibility that this will override some bytes that are important before the call?

Reply:
There is no problem, all test pass when switching to `mstore`